README.rst

```diff
@@ -1,12 +1,12 @@
 Scripts for generating and loading test xAPI events
-===================================================
+***************************************************
 
 |pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
 |license-badge| |status-badge|
 
 
 Purpose
-*******
+=======
 
 Some test scripts to help make apples-to-apples comparisons of different
 database backends for xAPI events. Supports direct database connections to
```

Comment:
Could you remove the references to MongoDB, Citus PostgreSQL from the text below?